integration/e2e/service.go

```diff
@@ -571,6 +571,32 @@ func (s *HTTPService) SumMetrics(metricNames ...string) ([]float64, error) {
 	return sums, nil
 }
 
+// Waits until promQL expression returns scalar (or vector with single value) with positive value.
+func (s *HTTPService) WaitForPromQL(matcher func(sums ...float64) bool, expr string) error {
```

Comment:
Not sure if the name of method is readable .. to me it reads like we wait for some evaluation only or somethig. 

What about `WaitForPromQLNonEmptyResult` :thinking: 
Not blocking, I get that we want short name as well.